Cargo.toml

```diff
@@ -11,6 +11,7 @@ exclude = ["assets/*", ".github"]
 approx = "0.5.1"
 byteorder = "*"
 chrono = "0.2.25"
+clap = "*"
```

Comment:
I don't know if this needs a follow up in the catkin build?

Reply:
nope